enterprise/backend/src/metabase_enterprise/workspaces/mirroring.clj

```diff
@@ -14,30 +14,33 @@
   #{:description :name :source :source_type})
 
 (defn- mirror-transform!
-  [database-id entities-info]
-  (let [;; At this point transform tables were created and `transform` that follows should have the `:mirror`
-        ;; key set. With addition of e.g. Clickhouse I expect some juggling with how we provide targets,
-        ;; but for now doing plain copy-paste schema name here.
-        ;; TODO (lbrdnk 2025-11-18) revisit schema, name vs db specific access pattern.
-        transform (-> entities-info :transforms first)
+  [database-id graph]
+  ;; With addition of e.g. Clickhouse I expect some juggling with how we provide targets,
+  ;; but for now doing plain copy-paste schema name here.
+  ;; TODO (lbrdnk 2025-11-18) revisit schema, name vs db specific access pattern.
+  (let [#_#_table-mapping (u/for-map [{:keys [name schema mapping]} (:outputs graph)]
+                                     [[name schema] ((juxt :name :schema) mapping)])
```

Comment:
We'll need to do mapping like this once we support multiple transforms. Mapping between tuples will be a bit annoying though.

I guess we could avoid it if we put the table details and mappings directly on the transforms from the start, but structurally I don't like that - in the flow chart we keep the data (isolate tables) and entities (mirror transforms) separate, and that feels more modular.